Text/PresidioPIIAnonymization/powerskill/app.py

```diff
@@ -0,0 +1,83 @@
+from typing import Dict, List
+
+import uvicorn
+from dotenv import load_dotenv
+from fastapi import FastAPI, Security, Depends, HTTPException
+from fastapi.security.api_key import APIKeyHeader, APIKey
+from objdict import ObjDict
+from pydantic import BaseModel
+from starlette.status import HTTP_400_BAD_REQUEST, HTTP_403_FORBIDDEN
+
+from powerskill import Presidio
+
+load_dotenv()
+app = FastAPI()
+presidio = Presidio()
```

Comment:
nit: Why not use the AnalyzerEngine and AnonymizerEngine directly? Might be easier for users to then customize

Reply:
It is more clean and clear IMO. Also, it is replica of the behavior separation which was done in the summary.